### ISSUE TITLE
MNIST handwritten digit recognition

### ISSUE THREAD

**Issue by adam-r-kowalski** [adam-r-kowalski/compute_graph#32]
Utilize a simple two layer neural network with sigmoid activation function and softmax activation on he last layer to build a multiclass classifier for MNIST.
